apps/bridge-dapp/src/components/EducationCard/EducationCard.tsx

```diff
@@ -0,0 +1,246 @@
+import { Close, HelpLineIcon, UsageGuideIcon } from '@webb-tools/icons';
+import { Button, Typography } from '@webb-tools/webb-ui-components';
+import cx from 'classnames';
+import { forwardRef, useState } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { EducationCardProps } from './types';
+import { Transition } from '@headlessui/react';
+
+const links = [
+  {
+    label: 'Usage Guide',
+    href: 'https://docs.webb.tools/docs/dapps/hubble-bridge/overview/',
+    Icon: UsageGuideIcon,
+  },
+  {
+    label: 'Get Started',
+    getLink: (tab: EducationCardProps['currentTab']) => {
+      return `https://docs.webb.tools/docs/dapps/hubble-bridge/usage-guide/${tab.toLowerCase()}/`;
+    },
+    Icon: UsageGuideIcon,
+  },
```

Comment:
These should be switched. Usage guides should be pointed to the dedicated operation (e.g. Deposit, Withdraw, Transfer), and Getting Started pointed to the Overview docs page.